Fineo/Common/Fineo.Interfaces/IMessageBus.cs

```diff
@@ -18,10 +18,14 @@ public interface IMessageBus
     {
         void Init(IMessageBusParams initParams);
 
+        string ID { get; set; }
+
         MessageBusDTO ReadNext();
 
         void Send(MessageBusDTO msg);
 
+        bool Delete();
```

Comment:
What the purpose

Reply:
Allows deleting of message queue (can be queue, topic, etc depending on implementation)